lib/database/models/typedefs/SequelizeRun.js

```diff
@@ -53,9 +53,18 @@
  * @property {string|null} pdpWorkflowParameters
  * @property {string|null} pdpBeamType
  * @property {string|null} readoutCfgUri
+ * @property {string|null} startOfDataTransfer
+ * @property {string|null} endOfDataTransfer
+ * @property {string|null} ctfFileCount
+ * @property {string|null} ctfFileSize
+ * @property {string|null} tfFileCount
+ * @property {string|null} tfFileCount
+ * @property {string|null} tfFileSize
+ * @property {string|null} otherFileCount
+ * @property {string|null} otherFileSize
+ * @property {number|null} runDuration
  * @property {string} createdAt
  * @property {string} updatedAt
- * @property {number|null} runDuration
```

Comment:
Please leave this here, I ordered fields to have them in the database order then the generated fields

Reply:
Done